tests/sandbox/create_message_batches/test_success.py

```diff
@@ -20,7 +20,15 @@
 @pytest.mark.parametrize('accept_headers', VALID_ACCEPT_HEADERS)
 def test_201_message_batch_valid_accept_headers(nhsd_apim_proxy_url, accept_headers):
     """
-    .. py:function:: Test valid accept headers
+    .. py:function:: Scenario: A user submitting a request with a \
+        valid accept header receives a 201 response
+
+        | **Given** the user provides a valid accept header
```

Comment:
**Given** the user provides a valid accept header when creating a batch of messages